Resources/Prototypes/XenoArch/artifact_triggers.yml

```
@@ -112,7 +112,7 @@
     groups:
       Acidic: [ Touch ]
     reactions:
-    - reagents: [ Blood, CopperBlood, InsectBlood, Slime, AmmoniaBlood, ZombieBlood ]
+    - reagents: [ Blood, CopperBlood, InsectBlood, Slime, AmmoniaBlood, ZombieBlood, BlackBlood  ]
```

YAMLlint (1.35.1)

[error] wrong indentation: expected 6 but found 4

(indentation)